src/modules/layout/components/HeaderNavigationBar.css

```diff
@@ -21,7 +21,12 @@
   text-transform: uppercase;
 }
 
-.navigation-item.active-navigation-item {
+.navigation-item.inactive-navigation-item {
+  opacity: 0.5;
```

Comment:
With a small viewport in Landkreis Alzey-Worms (who don't have news): When you open the LanguageDropDown, the title NEWS is shown above the languages. (Whyever this is not the case when you don't set opacity: 0.5; 🤔 )
You can fix this by switching the order of <HeaderNavigationBar /> and <HeaderActionBar /> in Header.js

Reply:
What do you mean by "switching the order"?

Reply:
the order in the html markup

Reply:
So the result would look like this in Header.js: 
``` jsx
    return (
      <Headroom scrollHeight={scrollHeight}>
        <header className={style.header}>
          <div className={style.logoWide}>
            <Link href={this.props.logoHref}>
              <img src={logoWide} />
            </Link>
          </div>
          <HeaderNavigationBar className={style.navigationBar} items={this.props.navigationItems} />
          <HeaderActionBar className={style.actionBar} items={this.props.actionItems} />
        </header>
      </Headroom>
    )
```

Reply:
Thanks, fixed it.